airflow/api_connexion/endpoints/task_instance_endpoint.py

```diff
@@ -716,7 +648,7 @@ def set_task_instance_note(
         error_message = f"Task Instance not found for dag_id={dag_id}, run_id={dag_run_id}, task_id={task_id}"
         raise NotFound(error_message)
 
-    ti, sla_miss = result
+    ti = result[0]
```

Comment:
if you change 
```
result = session.execute(query).one_or_none()
```
to 
```
result = session.scalar(query)
```
then now you can just do `ti=result`

or perhaps just call it ti in the first place

Reply:
that code change was what I was missing.  I futzed around trying to get just the ti object and it kept returning a one-item tuple.   I need more sqlalchemy practice.  Cheers.